lib/mysql-binuuid/type.rb

```diff
@@ -2,6 +2,12 @@ module MySQLBinUUID
   class InvalidUUID < StandardError; end
 
   class Type < ActiveModel::Type::Binary
+    attr_accessor :options
+
+    def initialize(opts = {})
+      self.options = opts
+    end
```

Comment:
I really like to have default options available, also as an opportunity to see which options there are and what their default values are.

```suggestion
    attr_reader :options

    DEFAULT_OPTIONS = {
      # By default an +InvalidUUID+ error is raised when a UUID is invalid. If you want to get a
      # nil value instead, register the attribute with `nil_invalid_values: true`.
      nil_invalid_values: false
    }.freeze

    def initialize(opts = {})
      @options = DEFAULT_OPTIONS.merge(opts)
    end
```